src/main/java/org/sonatype/repository/vgo/internal/util/VgoDataAccess.java

```diff
@@ -86,6 +93,11 @@ public Asset findAsset(final StorageTx tx, final Bucket bucket, final String ass
     return tx.findAssetWithProperty(MetadataNodeEntityAdapter.P_NAME, assetName, bucket);
   }
 
+  public Iterable<Asset> findPackageAssetsForModule(final StorageTx tx, final Repository repository, final String moduleName) {
+    Query query = Query.builder().where(format("name LIKE '%s/%%'", moduleName)).build();
```

Comment:
We should use a parameterized query here to prevent sql injection and escaping issues (e.g. https://github.com/sonatype/nexus-internal/blob/f1a057acd48746090bddcf47db6fe29da1e90ff7/components/nexus-repository/src/main/java/org/sonatype/nexus/repository/storage/Query.java)

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-vgo/pull/5/commits/7b3d4cba469b189fdd26126f5960ba8b9a1f0d26